coreneuron/nrniv/profiler_interface.h

```diff
@@ -37,6 +41,12 @@ struct Instrumentor {
     inline static void stop_profile() {
         std::initializer_list<int>{(TProfilerImpl::stop_profile(), 0)...};
     }
+    inline static void init_profile() {
```

Comment:
The `Instrumentor` static methods are missing documentation. I might be mistaken but it seems to me that `start_profile` static function serves the same purpose than the new `init_profile`, same for `stop_profile` / `finalize_profile`.

Reply:
Could you please take a look at c4c7a4f538baef7317eb7de4ee6250f47fa6047a?

Reply:
This commit is amazing! Thank you so much for improving this.